Explorer/Assets/DCL/Multiplayer/SDK/Systems/WriteAvatarEquippedDataSystem.cs

```diff
@@ -0,0 +1,102 @@
+        {
+            this.ecsToCRDTWriter = ecsToCRDTWriter;
+            this.componentPool = componentPool;
+        }
+
+        protected override void Update(float t)
+        {
+            HandleComponentRemovalQuery(World);
+            UpdateAvatarEquippedDataQuery(World);
+            CreateAvatarEquippedDataQuery(World);
+        }
+
+        [Query]
+        [None(typeof(PBAvatarEquippedData), typeof(DeleteEntityIntention))]
+        private void CreateAvatarEquippedData(in Entity entity, ref PlayerSDKDataComponent playerSDKDataComponent)
+        {
+            PBAvatarEquippedData pbComponent = componentPool.Get();
+            pbComponent.WearableUrns.Clear();
+
+            foreach (URN urn in playerSDKDataComponent.WearableUrns) { pbComponent.WearableUrns.Add(urn); }
+
+            pbComponent.EmoteUrns.Clear();
+
+            foreach (URN urn in playerSDKDataComponent.EmoteUrns) { pbComponent.EmoteUrns.Add(urn); }
+
+            ecsToCRDTWriter.PutMessage<PBAvatarEquippedData, PBAvatarEquippedData>(static (dispatchedPBComponent, pbComponent) =>
+            {
+                dispatchedPBComponent.WearableUrns.Clear();
+
+                foreach (URN urn in pbComponent.WearableUrns) { dispatchedPBComponent.WearableUrns.Add(urn); }
+
+                dispatchedPBComponent.EmoteUrns.Clear();
+
+                foreach (URN urn in pbComponent.EmoteUrns) { dispatchedPBComponent.EmoteUrns.Add(urn); }
+            }, playerSDKDataComponent.CRDTEntity, pbComponent);
+
+            World.Add(entity, pbComponent, playerSDKDataComponent.CRDTEntity);
+        }
+
+        [Query]
+        [None(typeof(DeleteEntityIntention))]
+        private void UpdateAvatarEquippedData(in Entity entity, ref PlayerSDKDataComponent playerSDKDataComponent, ref PBAvatarEquippedData pbComponent)
+        {
+            if (!playerSDKDataComponent.IsDirty) return;
+
+            pbComponent.WearableUrns.Clear();
+
+            foreach (URN urn in playerSDKDataComponent.WearableUrns) { pbComponent.WearableUrns.Add(urn); }
+
+            pbComponent.EmoteUrns.Clear();
+
+            foreach (URN urn in playerSDKDataComponent.EmoteUrns) { pbComponent.EmoteUrns.Add(urn); }
+
+            ecsToCRDTWriter.PutMessage<PBAvatarEquippedData, PBAvatarEquippedData>(static (dispatchedPBComponent, pbComponent) =>
+            {
+                dispatchedPBComponent.WearableUrns.Clear();
+
+                foreach (URN urn in pbComponent.WearableUrns) { dispatchedPBComponent.WearableUrns.Add(urn); }
+
+                dispatchedPBComponent.EmoteUrns.Clear();
+
+                foreach (URN urn in pbComponent.EmoteUrns) { dispatchedPBComponent.EmoteUrns.Add(urn); }
+            }, playerSDKDataComponent.CRDTEntity, pbComponent);
+
+            World.Set(entity, pbComponent);
+        }
+
+        [Query]
+        [All(typeof(PBAvatarEquippedData))]
```

Comment:
If you decide to keep this component (after my comments), considering it's pooled I believe it should be returned to the pool here: otherwise it will be never returned to the pool.
Please check another system as well.